gen3-client/g3cmd/download-multiple.go

```
@@ -158,9 +157,9 @@ func processOriginalFilename(downloadPath string, actualFilename string) string
 	}
 }
 
-func validateFilenameFormat(downloadPath string, filenameFormat string, rename bool, noPrompt bool) {
+func validateFilenameFormat(downloadPath string, filenameFormat string, rename bool, noPrompt bool) error {
```

if you wanna return an error then you should surface the error in [downloadFile](https://github.com/calypr/cdis-data-client/blob/efb18e72553ad2e47a0588821a8964de3e537ea1/gen3-client/g3cmd/download-multiple.go)

---

fixed